modules/mapper-extras/src/main/java/org/elasticsearch/index/mapper/extras/ScaledFloatFieldMapper.java

```diff
@@ -262,9 +263,14 @@ public Query rangeQuery(
             return (cache, breakerService) -> {
                 final IndexNumericFieldData scaledValues = new SortedNumericIndexFieldData.Builder(
                     name(),
-                    IndexNumericFieldData.NumericType.LONG
+                    IndexNumericFieldData.NumericType.LONG,
+                    null
```

Comment:
Tiny comment, maybe better to supply a non-null `ScriptField` that throws `UnsupportedOperationException`?

Reply:
Good suggestion! Done.